[PEAUTY-117] workspace address 필드 추가 및 로직 수정

harvartz opened the pull request:
## 📄 [PF-0] workspace address 필드 추가 및 로직 수정

Jira : [PEAUTY-XX](url)

## ✨ 변경 사항
- [x] 기능 추가/변경 설명
- [x] 버그 수정 설명
- [x] 문서 수정 설명

<!-- Pull Request의 설명을 추가하세요. -->
- workspace 필드 추가로 인한 수정

## 📅 작업 일정
<!-- 해당 작업을 수행하는데 예상했던 공수와 실제 소요되었던 공수를 기입해주세요. -->
- Expected MD: 0.1MD
- Actual MD: 0.1MD
### Difference reason (If correct, no need.)
- None.

## ✔️ 확인 사항

- [x] 코드가 잘 작동하는지 확인했나요?
- [x] 새로운 기능에 대한 테스트가 추가되었나요?
- [x] 문서가 업데이트되었나요?

harvartz committed:
fix: workspace address 추가

harvartz committed:
Merge branch 'release/v0.1.0' of https://github.com/peauty/peauty-BE into feature/PEAUTY-117

Shanate reviewed (changes requested):
큰 주소와 상세주소를 받을 수 있도록 변경해주시면 감사하겠습니다.

harvartz committed:
refactor: designer address -> workspace address 변경
- 기존의 designer address 삭제
- 모든 API 에 id 값 가져올 수 있게 진행(designer, workspace)
- DTO에 swagger example 코드 추가

Shanate reviewed (approved):
빠른 반영 감사합니다. 추가적으로 디자이너의 엔티티와 도메인에서 주소 삭제한 것도 좋아요
